doc/guide/build.hpp

```diff
@@ -249,11 +250,37 @@ suite.  You can build this component with
 $ make mlpack_test
 @endcode
 
-and then run all of the tests, or an individual test suite:
+We use Catch2 to write our tests
+To run all tests, you can simply run:
 
 @code
-$ bin/mlpack_test
-$ bin/mlpack_test -t KNNTest
+$ ./bin/mlpack_test
+@endcode
+
+To run all tests in a particular file you can run:
+
+@code
+$ ./bin/mlpack_test "[testname]"
+@endcode
+
+where testname is the name of the test suite. 
+For example to run all collaborative filtering tests implemented in cv_test.cpp you can run:
+
+@code
+./bin/mlpack_test "[CVTest]"
+@endcode
+
+Now similarly you can run all the binding related tests using:
+
+@code
+./bin/mlpack_test "[BindingTests]"
+@endcode
+
+To run a single test, you can explicitly provide the name of the test, for example, 
```

Comment:
```suggestion
To run a single test, you can explicitly provide the name of the test; for example, 
```
Just a pedantic grammar comment. :)